### ISSUE TITLE
Character limit too small for "to" field

### ISSUE THREAD

**Issue by elumbella** [neomutt/neomutt#2155]
## Expected Behaviour

When getting addresses from an outside source (in this case khard), all selected addresses should be included in the "to" field

## Actual Behaviour

When there is too many characters (probably around 1024), the remainder is cut off

## Steps to Reproduce

Use the khard interface with `khard email --parsable %s` as described [here](https://khard.readthedocs.io/en/latest/scripting.html#mutt) to search for several email addresses (it takes about 30 for me). Tag all of them, press enter.

## How often does this happen?

- Always (when addressing a longer list of recipients)

## When did it start to happen?

- Has always happened

## NeoMutt Version

```
Copyright (C) 1996-2016 Michael R. Elkins und andere.
NeoMutt übernimmt KEINERLEI GEWÄHRLEISTUNG. Starten Sie 'neomutt -vv', um
weitere Details darüber zu erfahren. NeoMutt ist freie Software.
Sie können es unter bestimmten Bedingungen weitergeben; starten Sie
'neomutt -vv' für weitere Details.

System: Linux 5.5.6-arch1-1 (x86_64)
ncurses: ncurses 6.2.20200212 (compiled with 6.1.20180127)
libidn2: 2.3.0 (compiled with 2.3.0)
GPGme: 1.13.1
libnotmuch: 5.2.0
hcache backends: kyotocabinet, gdbm, lmdb

Configure options: --prefix=/usr --sysconfdir=/etc --libexecdir=/usr/lib --gpgme --sqlite --autocrypt --lua --notmuch --gss --gnutls --sasl --with-ui=ncurses --with-idn2=/usr --disable-idn --idn2 --lmdb --kyotocabinet --gdbm

Compilation CFLAGS: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -std=c99 -fno-delete-null-pointer-checks -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -I/usr/include -DNCURSES_WIDECHAR

Standard Einstellungen:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash

Einstellungen bei der Kompilierung:
  +autocrypt +bkgdset +color +curs_set +fcntl -flock -fmemopen +futimens
  +getaddrinfo +gnutls +gpgme +gss +hcache -homespool +idn +inotify
  -locales_hack +lua +meta -mixmaster +nls +notmuch -openssl +pgp +sasl +smime
  +sqlite +start_color +sun_attachment +typeahead
MAILPATH="/var/mail"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

Um mehr über NeoMutt zu erfahren, besuchen Sie: https://neomutt.org
Falls Sie einen Bug in NeoMutt finden, eröffnen Sie bitte ein Ticket unter:
    https://github.com/neomutt/neomutt/issues
oder senden Sie eine E-Mail an: <neomutt-devel@neomutt.org>
```

## Extra Info

* Operating System and its version: Archlinux
* Were you using multiple copies of NeoMutt at once? No
* Were you using 'screen' or 'tmux'? No
* Is your email local (maildir) or remote (IMAP)? Local maildir

**Comment by gahr** [comment]
We actually provide 8k of room for addresses:
https://github.com/neomutt/neomutt/blob/418a038b0194dfece0edcf6f42fe5908196202a1/send.c#L210
This allows us to keep it on the stack just fine. What do you think a good default would be?

**Comment by elumbella** [comment]
I have no idea to be honest. I'm not a programmer so I cannot suggest any values. I only had trouble with a few email addresses not showing properly. I think you should be able to write to more than 20-30 people at once.

**Comment by gahr** [comment]
So 8192 / 30 = 273. I doubt your addresses are more than some 50 characters long, so something is odd here. I'll investigate more - perhaps we're not really using 8192?

**Comment by gahr** [comment]
I just tested, after accepting an address list of over 3000 character, the list is cut at 1024. Investigating.

**Comment by gahr** [comment]
Figured that out - fixed in #2191

**Comment by elumbella** [comment]
Thank you gahr for investigating and fixing. Much appreciated!
